pkgs/development/libraries/libvorbis/default.nix

```diff
@@ -16,6 +17,11 @@ stdenv.mkDerivation rec {
 
   doCheck = true;
 
+  preConfigure = if (stdenv.cc.isClang or false) then ''
```

Comment:
This could be `lib.optionalString (stdenv.cc.isClang or false) ''…''`.

Reply:
Done, thanks!